reV/supply_curve/supply_curve.py

```diff
@@ -612,61 +656,85 @@ def _compute_trans_cap_cost(trans_table, trans_costs=None,
-                cost = np.hstack(cost)
         else:
-            feature = TC(trans_table, line_limited=line_limited,
-                         **trans_costs)
             cost = []
-            for _, row in trans_table.iterrows():
-                if connectable:
-                    capacity = row['capacity']
-                else:
-                    capacity = None
+            for sc_gid, sc_table in groups:
+                capacity = cls._get_capacity(sc_gid, sc_table,
+                                             connectable=connectable)
+                cost.append(TC.feature_costs(sc_table,
+                                             capacity=capacity,
+                                             avail_cap_frac=avail_cap_frac,
+                                             line_limited=line_limited,
+                                             **trans_costs))
+
+        cost = np.hstack(cost).astype('float32')
+        logger.info('LCOT cost calculation is complete.')
 
-                tm = row.get('transmission_multiplier', 1)
-                cost.append(feature.cost(row['trans_gid'], row['dist_km'],
-                                         capacity=capacity,
-                                         transmission_multiplier=tm))
+        return cost
 
-            cost = np.array(cost, dtype='float32')
+    def compute_total_lcoe(self, fcr, transmission_costs=None,
+                           avail_cap_frac=1, line_limited=False,
+                           connectable=True, max_workers=None,
+                           consider_friction=True):
+        """
+        Compute LCOT and total LCOE for all sc point to transmission feature
+        connections
 
-        logger.info('LCOT cost calculation is complete.')
+        Parameters
+        ----------
+        fcr : float
+            Fixed charge rate, used to compute LCOT
+        transmission_costs : str | dict, optional
+            Transmission feature costs to use with TransmissionFeatures
+            handler: line_tie_in_cost, line_cost, station_tie_in_cost,
+            center_tie_in_cost, sink_tie_in_cost, by default None
+        avail_cap_frac : int, optional
+            Fraction of transmissions features capacity 'ac_cap' to make
+            available for connection to supply curve points, by default 1
+        line_limited : bool, optional
+            Flag to have substation connection is limited by maximum capacity
+            of the attached lines, legacy method, by default False
+        connectable : bool, optional
+            Flag to only compute tranmission capital cost if transmission
+            feature has enough available capacity, by default True
+        max_workers : int | NoneType, optional
+            Number of workers to use to compute lcot, if > 1 run in parallel.
+            None uses all available cpu's. by default None
+        consider_friction : bool, optional
+            Flag to consider friction layer on LCOE, by default True
```

Comment:
Could we change this docstring to "Flag to consider friction layer on LCOE when "mean_lcoe_friction" is in the sc points input, by default True"